tools/subgraph-example/hardhat.config.ts

```diff
@@ -62,6 +65,88 @@ task("mintERC721", "Mints an ERC721 token to a recipient", async (taskArgs, hre)
   await mintNFT(<string>process.env.RECEIVER_PRIVATE_KEY, hre);
 });
 
+task("deployHTS", "Deploys HTS Fungible Token", async (taskArgs, hre) => {
+  const wallet = new Wallet(
```

Comment:
Q: can you use the `createWallet()` here like is done above?

Reply:
createWallet func here uses ethers, but we need to use SDK to make HAPI operations, to test those synthetic events

Reply:
is not possible to deploy a HTS token using system contracts? hence through the RPC EVM and ethers js?

Reply:
Understood. @georgi-l95 can you add a comment so it's clear you're using the SDK to carry out HTS transactions that don't pass through  system contract logic.

In general we should use ethers except for situations like this or cases where we need to specficy Hedera details not exposed by the systems contract methods.

Reply:
> is not possible to deploy a HTS token using system contracts? hence through the RPC EVM and ethers js?

I think this falls under the category of hts precompile events, which is something else and it's not supported https://github.com/hashgraph/hedera-services/issues/4127

Reply:
Thanks for the explanation! 👍